brand/schema.py

```diff
@@ -255,11 +255,16 @@ class Query(graphene.ObjectType):
     commentary = relay.Node.Field(Commentary)
     commentaries = DjangoFilterConnectionField(Commentary)
 
-    brand = graphene.Field(Brand, tag=graphene.Argument(graphene.String, required=True))
+    brand_by_tag = graphene.Field(Brand, tag=graphene.Argument(graphene.String, required=True))
 
-    def resolve_brand(root, info, tag):
+    brand_by_name = graphene.Field(Brand, name=graphene.Argument(graphene.String, required=True))
+
+    def resolve_brand_by_tag(root, info, tag):
```

Comment:
Same as above - better to leave this endpoint named 'brand' unless you are also updating all the code in other repos that are relying on it.